### ISSUE TITLE
Renewal interface for expiring hosts

### ISSUE THREAD

**Issue by GoogleCodeExporter** [openipam/openipam#6]
```
Purpose of this enhancement, including design decisions:

In Manage Hosts, there needs to be an interface to renew hosts that are
about to expire. This interface should also allow for mass renewal of hosts.

Process:

Make it so!

Special factors to remember:

This renewal interface depends deeply on issue #2, the Django refactor. If
we move forward with that, then it makes sense that these changes are
merged into the refactor of the UI for managing hosts. This issue also
depends on issue #5, bulk host management, and could be tied into that.

Also, other interfaces need to be able to filter to expiring hosts, so
remember low coupling with backend changes. 

```

Original issue reported on code.google.com by `fotinaki...@gmail.com` on 27 Jan 2009 at 6:48
